src/ml_gym/frontend/dashboard/src/features/experimentConfig/experimentConfigSlice.tsx

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { ExperimentConfigMessageCollectionType, ExperimentConfigType} from '../../app/datatypes'
+import type { RootState } from '../../app/store';
+
+const initialState = {
+    messages: [],
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=10020887)
```suggestion
  messages: [],
```